### DISCUSSION TITLE
Add Jalali Date into Core

### DISCUSSION THREAD

**Discussion by dn1353** [YetiForceCompany/YetiForceCRM#14194] [Feature improvement]
Hi Dear Developers,

Would you please add support Jalali Date (Persian Date) into Yetiforce Core? 
in other words, when we set Language in Persian(Farsi) in Yetiforce, the system automatically sets the Jalali date as a base calendar system. 
I know that Yetiforce uses https://fullcalendar.io as a calendar system.
For better understanding please go below link and set locales-->fa then you can see the full support of Jalalidate. 

https://fullcalendar.io/#demos
locales-->fa

Kind regards
Davoud

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->

### Describe the solution you'd like

Would you please add support Jalali Date (Persian Date) into Yetiforce Core? 
in other words, when we set Language in Persian(Farsi) in Yetiforce, the system automatically sets the Jalali date as a base calendar system. 
I know that Yetiforce uses https://fullcalendar.io as a calendar system.
For better understanding please go below link and set locales-->fa then you can see the full support of Jalalidate. 

https://fullcalendar.io/#demos
locales-->fa

Kind regards
Davoud
